src/play.html

```diff
@@ -161,7 +161,7 @@
       }
 
       .tab-icon {
-          left: -0.5em;
```

Comment:
What's the intent behind this change? To account for the slight shift in the margin?

Reply:
That's right! The tab icon was being positioned relative to the inside of the margin, so to make up for the 8px margin, it was being shifted back 8px (0.5em is equivalent in most browsers by default if you don't change the font size). So now that the margin is gone, it doesn't need to be shifted. Without this, the little pull out tab thing doesn't stick out enough.